pkg/virt-controller/services/template_test.go

```diff
@@ -882,6 +882,23 @@ var _ = Describe("Template", func() {
 				Expect(value).To(Equal("k6t-eth0"))
 			})
 		})
+		Context("With Istio sidecar.istio.io/inject annotation", func() {
+			It("should mount default serviceAccountToken", func() {
+				vmi := v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "testvmi",
+						Namespace: "default",
+						UID:       "1234",
+						Annotations: map[string]string{
+							ISTIO_INJECT_ANNOTATION: "true",
+						},
+					},
+				}
+				pod, err := svc.RenderLaunchManifest(&vmi)
```

Comment:
this is me being extremely picky, you can disregard this comment; it would never the allow the reader to be more focused if you'd create the `vmi` and `pod` objects in the test setup stage - `BeforeEach` - under this context.

Reply:
Yep. I like your idea. Done, thanks!